Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -612,6 +608,30 @@ protected theorem repr_reindex (b : OrthonormalBasis ι 𝕜 E) (e : ι ≃ ι')
 
 end OrthonormalBasis
 
+namespace EuclideanSpace
+
+variable (𝕜 ι)
+
+/-- The basis `Pi.basisFun`, bundled as an orthormal basis of `EuclideanSpace 𝕜 ι`. -/
+noncomputable def basisFun :
+    OrthonormalBasis ι 𝕜 (EuclideanSpace 𝕜 ι) := ⟨LinearIsometryEquiv.refl _ _⟩
+
+@[simp]
+theorem basisFun_repr (x : EuclideanSpace 𝕜 ι) (i : ι) :
+    (basisFun ι 𝕜).repr x i = x i := rfl
```

Comment:
Can you add the lemma about basisFun ι 𝕜 i` too

Reply:
I added the version that's defeq to `Pi.basisFun ι 𝕜 i`. Another possibility would be to defeq to `PiLp.basisFun` but I think it is better to go directly to the simplest expression.